nixos/modules/services/web-servers/apache-httpd/per-server-options.nix

```diff
@@ -24,13 +24,23 @@ with lib;
     '';
   };
 
-  port = mkOption {
-    type = types.int;
-    default = 0;
-    description = ''
-      Port for the server.  0 means use the default port: 80 for http
-      and 443 for https (i.e. when enableSSL is set).
-    '';
+  listen = mkOption {
+     type = types.listOf (types.submodule (
+          {
+            options = {
+              port = mkOption {
+                type = types.int;
```

Comment:
Good question. Either you use the default (see getListen), or you're able to set port as well if you deviate from standard setup that much. That's why I didn't care. Which default should be choosen?